dask_sql/server/responses.py

```diff
@@ -64,7 +64,9 @@ def __init__(self, request: Request, next_url: str = None, cancel_url: str = Non
 class DataResults(QueryResults):
     @staticmethod
     def get_column_description(df):
-        sql_types = [str(python_to_sql_type(t)) for t in df.dtypes]
+        sql_types = [
+            str(python_to_sql_type(t).getSqlType()).partition(".")[2] for t in df.dtypes
```

Comment:
Ideally we should add a `toString` method to either `DaskTypeMap` or `SqlTypeName`, but I'm interested in if we can define a dunder method through pyo3 so that the original code would just work here?

Reply:
It looks like this might be doable? https://pyo3.rs/main/class/object.html?highlight=__str__#string-representations